keymanapp/keyman
# chore(common): Check in crowdin strings for Kibaku 📜

This checks in the crowdin strings for the locale `ckl` for Kibaku. Does not include macOS.

- [x] TODO: @sgschantz to check on adding ckl (ckl-NG?) locale in XCode

## User Testing
For the listed platforms:
1. Load the PR build
2. Set the UI/locale to Kibaku
3. Verify the UI is Kibaku
Note: The crowdin translator left some strings in English, so that's out of our control

* **TEST_ANDROID**
1. Verify Kibaku strings are like the screenshot
![kibaku](https://user-images.githubusercontent.com/7358010/146119415-71c6c201-5c35-4b9f-94d2-3f9215ab05fc.png)

* **TEST_IOS** - Wait until Shawn addresses the XCode TODO before testing

* **TEST_LINUX**
1. start Keyman configuration with: `LANGUAGE=ckl_NG km-config`

* **TEST_WINDOWS**
1. start Keyman configuration and select UI language 'Nya Nasara'
